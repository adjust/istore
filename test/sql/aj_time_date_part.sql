SELECT date_part('hour','2029-01-01 00:20:20'::aj_time);
SELECT date_part('hour','2029-01-01 01:20:20'::aj_time);
SELECT date_part('hour','2029-01-01 02:20:20'::aj_time);
SELECT date_part('hour','2029-01-01 03:20:20'::aj_time);
SELECT date_part('hour','2029-01-01 04:20:20'::aj_time);
SELECT date_part('hour','2029-01-01 05:20:20'::aj_time);
SELECT date_part('hour','2029-01-01 06:20:20'::aj_time);
SELECT date_part('hour','2029-01-01 07:20:20'::aj_time);
SELECT date_part('hour','2029-01-01 08:20:20'::aj_time);
SELECT date_part('hour','2029-01-01 09:20:20'::aj_time);
SELECT date_part('hour','2029-01-01 10:20:20'::aj_time);
SELECT date_part('hour','2029-01-01 11:20:20'::aj_time);
SELECT date_part('hour','2029-01-01 12:20:20'::aj_time);
SELECT date_part('hour','2029-01-01 13:20:20'::aj_time);
SELECT date_part('hour','2029-01-01 14:20:20'::aj_time);
SELECT date_part('hour','2029-01-01 15:20:20'::aj_time);
SELECT date_part('hour','2029-01-01 16:20:20'::aj_time);
SELECT date_part('hour','2029-01-01 17:20:20'::aj_time);
SELECT date_part('hour','2029-01-01 18:20:20'::aj_time);
SELECT date_part('hour','2029-01-01 19:20:20'::aj_time);
SELECT date_part('hour','2029-01-01 20:20:20'::aj_time);
SELECT date_part('hour','2029-01-01 21:20:20'::aj_time);
SELECT date_part('hour','2029-01-01 22:20:20'::aj_time);
SELECT date_part('hour','2029-01-01 23:20:20'::aj_time);
SELECT date_part('month','2029-01-01 23:20:20'::aj_time);
