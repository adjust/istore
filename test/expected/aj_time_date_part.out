SELECT date_part('hour','2029-01-01 00:20:20'::aj_time);
 date_part 
-----------
         0
(1 row)

SELECT date_part('hour','2029-01-01 01:20:20'::aj_time);
 date_part 
-----------
         1
(1 row)

SELECT date_part('hour','2029-01-01 02:20:20'::aj_time);
 date_part 
-----------
         2
(1 row)

SELECT date_part('hour','2029-01-01 03:20:20'::aj_time);
 date_part 
-----------
         3
(1 row)

SELECT date_part('hour','2029-01-01 04:20:20'::aj_time);
 date_part 
-----------
         4
(1 row)

SELECT date_part('hour','2029-01-01 05:20:20'::aj_time);
 date_part 
-----------
         5
(1 row)

SELECT date_part('hour','2029-01-01 06:20:20'::aj_time);
 date_part 
-----------
         6
(1 row)

SELECT date_part('hour','2029-01-01 07:20:20'::aj_time);
 date_part 
-----------
         7
(1 row)

SELECT date_part('hour','2029-01-01 08:20:20'::aj_time);
 date_part 
-----------
         8
(1 row)

SELECT date_part('hour','2029-01-01 09:20:20'::aj_time);
 date_part 
-----------
         9
(1 row)

SELECT date_part('hour','2029-01-01 10:20:20'::aj_time);
 date_part 
-----------
        10
(1 row)

SELECT date_part('hour','2029-01-01 11:20:20'::aj_time);
 date_part 
-----------
        11
(1 row)

SELECT date_part('hour','2029-01-01 12:20:20'::aj_time);
 date_part 
-----------
        12
(1 row)

SELECT date_part('hour','2029-01-01 13:20:20'::aj_time);
 date_part 
-----------
        13
(1 row)

SELECT date_part('hour','2029-01-01 14:20:20'::aj_time);
 date_part 
-----------
        14
(1 row)

SELECT date_part('hour','2029-01-01 15:20:20'::aj_time);
 date_part 
-----------
        15
(1 row)

SELECT date_part('hour','2029-01-01 16:20:20'::aj_time);
 date_part 
-----------
        16
(1 row)

SELECT date_part('hour','2029-01-01 17:20:20'::aj_time);
 date_part 
-----------
        17
(1 row)

SELECT date_part('hour','2029-01-01 18:20:20'::aj_time);
 date_part 
-----------
        18
(1 row)

SELECT date_part('hour','2029-01-01 19:20:20'::aj_time);
 date_part 
-----------
        19
(1 row)

SELECT date_part('hour','2029-01-01 20:20:20'::aj_time);
 date_part 
-----------
        20
(1 row)

SELECT date_part('hour','2029-01-01 21:20:20'::aj_time);
 date_part 
-----------
        21
(1 row)

SELECT date_part('hour','2029-01-01 22:20:20'::aj_time);
 date_part 
-----------
        22
(1 row)

SELECT date_part('hour','2029-01-01 23:20:20'::aj_time);
 date_part 
-----------
        23
(1 row)

SELECT date_part('month','2029-01-01 23:20:20'::aj_time);
ERROR:  unknown time part to extract
